index.html

```diff
@@ -7043,11 +7046,23 @@ <h2>ISO TC184/SC4 (Sebastian/Kaz)</h2>
     </section>
 
     <section id="edgex">
-      <h2>EdgeX (McCool)</h2>
-      <section id="liaison-6" class="ednote">
-        TODO: Brief description of liaised SDO, work areas.
-        Scope of the collaboration.
-        Provide Website link.
+      <h2>EdgeX Foundry</h2>
+      <p>
+        The EdgeX Foundry [EDGEX] is a community-driven open source project,
+        organized under the Linux Foundation, to build
+        a secure software stack for IoT hubs. Its goal is to enable interoperability
+        by combining a set of key IoT services with a set of interfaces to a variety
+        of IoT device protocols and ecosystems.  This organization focuses on software
+        development rather than standards development per se.  
```

Comment:
I'd say the EdgeX goal is to create a reference architecture. The developed software is a reference implementation of that architecture.

Reply:
I'm going to steal your text and change the last sentence to: "The EdgeX goal is to create a reference architecture, and there is a reference implementation of that architecture."

Reply:
made a few more changes than just changing the last sentence to make things consistent, but I think I've captured your point and am going to mark this resolved.